ui-ngx/src/app/shared/components/js-func.component.html

```diff
@@ -45,9 +45,8 @@
     <fieldset style="width: initial">
       <div matTooltip="{{(fullscreen ? 'fullscreen.exit' : 'fullscreen.expand') | translate}}"
            matTooltipPosition="above"
-           style="border-radius: 50%"
-           (click)="fullscreen = !fullscreen">
-        <button type='button' mat-icon-button class="tb-mat-32">
+           style="border-radius: 50%">
+        <button type='button' mat-icon-button class="tb-mat-32" [disabled]="disabled" (click)="fullscreen = !fullscreen">
```

Comment:
Why did you need to disable the ability to open the full-screen editor? Ex. for better content readin